docs/index.md

```diff
@@ -106,6 +106,12 @@ Use the the `irida-uploader.sh` script included with the source code to upload.
 
 `./irida-uploader.sh /path/to/the/sequencing/run/`
 
+
+#### Note:
+After uploading, an `irida_uploader_status.info` file will be created which indicates if a run is complete, or has failed
+
+You can edit this file to set a run's status to `new` to reupload, or use the `--force` option when running the uploader to ignore the status of a run directory.
```

Comment:
I would actually say users should completely delete this file if they wish to re-upload. The reason being is that we've now added the **run id** to the status file.  So, changing the status to `new` could lead people to believe that it will still upload to the same sequencing run id.

Reply:
That's not accurate. I've made it so the run_id is only included on a successful upload, partial uploads and uploads that error out do not include the run id. There is no functionality to continue an upload with an existing run id in this new uploader.

I have changed it to say `delete this file` now though

Reply:
Thanks.

Can you change *You can delete this file make it ready for* to *You can delete this file __to__ make it ready for*?

Reply:
Oops, fixed now.